src/server/main.cc

```diff
@@ -11,14 +11,40 @@
 #include "src/proto/protocached.pb.h"
 #include "src/proto/protocached.grpc.pb.h"
 
-void RunServer() {
+// TODO(joejoevictor): Move this to separate header and implementation file
+class ProtocachedServiceImpl final : public protocached::Protocached::Service {
+  grpc::Status Get(
+    grpc::ServerContext* context,
+    const ::protocached::GetRequest* request,
```

Comment:
why is there :: in front of the protocached namespace for GetRequest, but not for GetResponse? Is there any reason for it? I don't understand... (genuinely)

Reply:
Oh, sorry I left that there. It should have been removed. A namespace starts with :: just means it's an "absolute" namespace(or global namespace).

Maybe we should just use absolute namespace everywhere? What do you think?